.gitignore

```diff
@@ -11,6 +11,11 @@ node_modules
 notes
 
 $ Ignore the .env file for the server
-.env.example
+.env
 
-dist
\ No newline at end of file
+# Ignore local logs
+access.log
+trustwallet.log
+
+dist
+package-lock.json
```

Comment:
What's the deal with `package-lock.json` it sounds like they recommend to commit this one

Reply:
Official docs controversial https://docs.npmjs.com/files/package-lock.json.  I thought to force it to be ignored.